etstool.py

```diff
@@ -122,12 +122,10 @@
 }
 
 toolkit_dependencies = {
-    "pyside": {"pyside"},
-    # XXX once pyside2 is available in EDM, we will want it here
+    # XXX once pyside2 is available in EDM, we will want it here. For now
+    # we do a pip install.
     "pyside2": set(),
-    "pyqt": {"pyqt<4.12"},  # FIXME: build of 4.12-1 appears to be bad
-    # XXX once pyqt5 is available in EDM, we will want it here
-    "pyqt5": set(),
+    "pyqt5": {"pyqt5"},
```

Comment:
This mapping will be inconsistent with traitsui and pyface, where the mapping is `"pyqt": {"pyqt5"}` instead. It might add to the cognitive load for developers working across multiple ETS projects if envisage chooses a different naming.

I too preferred "pyqt5" in the toolkit name, but I deemed that opinion subjective. Afterall, this is only affecting developers' tools so this is easy to change one way or another.

Reply:
Yes, that's a good point. I'll change it.  But in that case, for consistency, we should probably be using `pyside` instead of `pyside2`. :-)

Reply:
Actually, I think there _is_ a good non-subjective reason to prefer `pyqt5` over `pyqt`, which is that it matches the names of the corresponding EDM and PyPI packages.

Reply:
I'm going to stick with the existing `pyqt5`, both because it matches the previous behaviour on this repo (I'd rather not switch `pyqt` from being valid and meaning one thing to still being valid and meaning something different) and because it matches the PyPI and EDM names.